.env

```diff
@@ -1 +1,2 @@
 PORT=8080
+DISABLE_ESLINT_PLUGIN=true
```

Comment:
I know it's probably getting annoying but newline please :(

Reply:
oh my bad, that was my fault